doc/libnetconf.doc

```diff
@@ -390,6 +391,13 @@
  * Another option for authorized clients is to reference another endpoint's clients, however be careful not to create a cyclic reference
  * (see ::nc_server_config_add_ssh_endpoint_client_ref()). An authorized client MUST authenticate to all of it's configured authentication methods.
  *
+ * The Keyboard Interactive authentication method is also supported. There are three options.
+ * The first is to use Linux PAM. You need to specify the service name using ::nc_server_ssh_set_pam_conf_filename().
+ * The second is to use the system's locally configured user's. The only Keyboard Interactive challenge will be the given
+ * user's password (that is if he's found on the system).
+ * The last option is to define your own callback to perform the authentication, see ::nc_server_ssh_set_interactive_auth_clb().
+ * The callback has a higher priority than the other two methods.
+ *
```

Comment:
Explanation of each implementation is missing.

Line 395: If `libpam` is found, Linux PAM is used.
Line 396: The second option is using local users if the standard functions for accessing them are found.